### PR TITLE
Data generation

### PR DESCRIPTION
**Pull request by H-T-Smith**
Created functions for random data generation for all tables in the database.

### CODE REVIEW

File: data_generation.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+		else:
+			maxID = maxID[0]
+
+		if type(maxID) == type(None):
+			maxID = 0
+		return maxID+1
+
+def find_id(tableID, table):
+	with sf.create_connection('database.db') as conn:
+		firstID = sf.execute_statement
```
Comment (ShawnButler2021) [inline data_generation.py]: This execute_statement should have something in it, right?

File: sql_script.sql [hunk 2]
```diff
@@ -18,7 +18,7 @@ CREATE TABLE inventory (
     itemID INT PRIMARY KEY,
     userID INT NOT NULL,
     analyticsID INT NOT NULL,
-    itemName VARCHAR(20) NOT NULL
```
Comment (ShawnButler2021) [inline sql_script.sql:21]: Use the current sql_script. A new one will be uploaded soon, whose only change will be in orders

File: data_generation.py (hunk 1 above)
Comment (H-T-Smith) [reply data_generation.py]: This was on my in-progress commit, and was removed.

File: sql_script.sql (hunk 2 above)
Comment (H-T-Smith) [reply sql_script.sql]: Reverted.

**Pull request merged**